go.mod

```diff
@@ -1,15 +1,12 @@
 module github.com/ONSdigital/dp-authorisation/v2
 
-go 1.19
-
-// to avoid [CVE-2023-48795] CWE-354: Improper Validation of Integrity Check Value
-replace golang.org/x/crypto => golang.org/x/crypto v0.17.0
+go 1.21
```

Comment:
If this is being upgraded, doesn't that mean that it should also be updated in places like the `ci/*.yml` files? And possibly other places too.